menas/src/test/scala/za/co/absa/enceladus/menas/auth/jwt/JwtFactoryTest.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.menas.auth.jwt
+
+import io.jsonwebtoken.security.WeakKeyException
+import org.scalatest.{Matchers, WordSpec}
+
+class JwtFactoryTest extends WordSpec with Matchers {
+
+  private val secret = "1234567890qwertyuiopasdfghjklzxcvbnm"
+  private val jwtFactory = new JwtFactory(secret)
+
+  "JwtFactory" should {
+    "throw an exception" when {
+      "creating a JwtBuilder with a weak secret" in {
+        intercept[WeakKeyException] {
+          new JwtFactory("NotEnoughBits").jwtBuilder()
+        }
+      }
+      "creating a JwtParser with a weak secret" in {
```

Comment:
Isn't this the same code as above?

Reply:
It checks that the `JWT Parser` the above one checks the `JWT Builder`

Reply:
Blind me, sorry.